src/Scan.js

```diff
@@ -376,6 +390,7 @@ class Scan extends React.Component {
       <ConfirmStatusModal
         open={!!request}
         onConfirm={this.onModalClose}
+        onCancel={this.handleOnAfterPrint}
```

Comment:
Are you sure that you passed this callback in all necessary places? What about outeForDeliveryModal?

Reply:
RouteForDeliveryModal has prop onClose={this.onModalClose}. And in this.onModalClose focus is setting in